src/pspm_compute_visual_angle.m

```diff
@@ -93,92 +89,75 @@
     if strcmpi(options.eyes, 'all') || strcmpi(options.eyes(1), eye)
         gaze_x = ['gaze_x_', eye];

```

Comment:
So `eye` variable here should be `l` or `r` since the name of the channel can only be `gaze_x_l`, gaze_y_r`, etc.